packages/RadioCard/__tests__/RadioCard.spec.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import Box from '@tds/core-box'
+import Paragraph from '@tds/core-paragraph'
+import RadioCard from '../RadioCard'
+
+describe('RadioCard', () => {
+  const defaultProps = {
+    label: 'The radio',
+    name: 'radio_group',
+    value: 'the-value',
+    width: 200,
```

Comment:
@janeszelag can you remove the height and width properties in the tests, and pass in all appropriate props please. Thanks!

Reply:
done :)